test/Spago/Bundle.purs

```diff
@@ -2,54 +2,94 @@ module Test.Spago.Bundle where
 
 import Test.Prelude
 
+import Data.String as Str
+import Spago.Command.Bundle (checkWatermarkMarkerFileName)
+import Spago.FS as FS
+import Spago.Generated.BuildInfo as BuildInfo
 import Test.Spec (Spec)
 import Test.Spec as Spec
+import Test.Spec.Assertions (shouldNotEqual)
+import Test.Spec.Assertions.String (shouldStartWith)
 
 spec :: Spec Unit
 spec = Spec.around withTempDir do
-  Spec.describe "bundle" do
+  Spec.focus $ Spec.describe "bundle" do
```

Comment:
```suggestion
  Spec.describe "bundle" do
```

Reply:
No worries - I have merged those in quite a few times already, so extra sensitive to them now.

I wonder how we could prevent merging those in at all - maybe spec should allow for focusing using environment variables like many other test runners do, that would remove the need to edit the source at all

Reply:
Yes please! I have been wishing for this in spec for some years already, but my todo list always ends up too long 😄